content/academy/data_collection_basics/using_devtools.md

```diff
@@ -32,72 +32,73 @@ The function to do that is called `document.querySelector('some-selector')` and
 
 > You can find available CSS selectors and their syntax on the <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Selectors" target="_blank">MDN CSS Selectors page</a>.
 
-At the time of writing, the HTML element that contained all the data we wanted had a `<div>` tag and a `tr site-listing` class. This actually means that there were two classes applied to the element. `tr` and `site-listing`. The `site-listing` class looked like the more reliable pick for targeting the element so we chose that one. We ran this command in the DevTools Console to select the element we wanted.
+At the time of writing, the HTML element that contained all the data we wanted had an `<a>` tag and a `animated fadeIn` class, plus an `aria-label` attribute. This actually means that there were two classes applied to the element - `animated` and `fadeIn`. Neither of these seem reliable classes to go off of; however, each element also includes an `href` attribute which includes `/product/`, which we can use to our advantage.
 
-```js
-document.querySelector('div.site-listing');
+```JavaScript
+document.querySelector('a[aria-label][href*="/product/"]');
```

Comment:
Please add more explanation or a link to learn more about the href selector. Also, is the aria-label needed? I see that later we're working without it.

Reply:
it's not necessary, I'll take it out